core/src/main/groovy/noe/tomcat/configure/SecureHttpConnectorTomcat.groovy

```diff
@@ -1,4 +1,10 @@
 package noe.tomcat.configure
+
+import noe.common.utils.PathHelper
+import noe.common.utils.Platform
+
+import java.nio.file.Path
```

Comment:
Not sure whether I checking the most fresh versions but the import is still here.